package.json

```diff
@@ -17,12 +17,14 @@
     "@types/lodash": "^4.14.194",
     "@types/sinon": "^10.0.15",
     "babel-jest": "^29.4.1",
+    "crypto-browserify": "^3.12.0",
     "eslint": "^8.33.0",
     "husky": "^8.0.0",
     "jest": "^29.4.1",
     "prettier": "^2.0.5",
     "pretty-quick": "^2.0.1",
     "sinon": "^15.1.0",
+    "stream-browserify": "^3.0.0",
```

Comment:
I actually couldn't get the SDK to build locally. Looks like these libs needed to be shimmed